ansible_builder/utils.py

```diff
@@ -69,7 +70,13 @@ def run_command(command, capture_output=False, allow_error=False):
                                    stdout=subprocess.PIPE,
                                    stderr=subprocess.STDOUT)
     except FileNotFoundError:
-        logger.error(f"You do not have {command[0]} installed, please specify a different container runtime for this command.")
+        msg = f"You do not have {command[0]} installed."
+        if command[0] in constants.runtime_files:
+            msg += "\nPlease specify a different --container-runtime."
+            msg += "\nOption avaibility: {0}.".format(
+                ', '.join([f'{runtime}: {bool(shutil.which(runtime))}' for runtime in constants.runtime_files])
```

Comment:
@chrismeyersfsu I don't want to get bogged down in grammar, current demo:

```
You do not have podman installed.
Please specify a different --container-runtime.
Detected installed: podman: False, docker: True.
```

I will change this wording, if you will tell me _exactly_ what it should say, because it's going to be really easy to second-guess and iterate on a million tiny changes.

Reply:
You do not have podman installed.
podman is the default container runtime. Please either install podman or specify an alternative supported container runtime by passing `--container-runtime` on the command line. Below are the list of supported container runtime's and whether or not they were found on your system.
podman not installed
docker installed

Reply:
> podman is the default container runtime.

That doesn't make sense for this error. It could have been the default, or it could have been the user-provided value by `--container-runtime=podman` (or repeat this situation for docker). The default and the value we're trying to use are not the same things.

I'll delete that sentence and see how it looks.

Reply:
```
You do not have docker installed.
Please either install docker or specify an alternative container runtime by passing --container-runtime on the command line.
Below are the supported container runtimes and whether or not they were found on your system.
podman: not installed, docker: not installed
```

with latest commit, tested.